### ISSUE TITLE
Mapping packages do not show any output (geemap, mapview, leaflet ...)

### ISSUE THREAD

**Issue by Cidree** [posit-dev/positron#4057]
## Positron Version: 1.91.0
b189a8b9c77a1fcc8a79732bfe24b2fa018436cc
x64

## Steps to reproduce the issue:

```py
import ee
import geemap

ee.Initialize()

m = geemap.Map()
m
```

## What did you expect to happen?

To display a map.

## Were there any error messages in the Output panel or Developer Tools console?

There are no error messages. When I execute the previous code, I got this result in the console:

```
>>> m
Map(center=[0, 0], controls=(WidgetControl(options=['position', 'transparent_bg'], widget=SearchDataGUI(childr…
```

But no map is shown in the plots or viewer panes. I have similar issues with several R packaes:

**mapview** and **leaflet**

The following code:

```r
library(mapview)
mapview()

library(leaflet)
leaflet()
```

Displays a white window in the plots panel, and the following output in the console:

```
> mapview(aa)
<R HTML Widget>

> leaflet()
<leaflet HTML widget>
```

**Comment by juliasilge** [comment]
For me, I see this code behaving the same way in RStudio and Positron:

```r
library(leaflet)
leaflet()
```

In the mapview example, we are not correctly loading HTML dependencies into the Plots pane and there are errors in the Developer Tools console about not loading leaflet providers with `net::ERR_ACCESS_DENIED`.

Related to #3756, #3968, and #2023

**Comment by jmcphers** [comment]
mapview should work now, after #4151: 

<img width="1389" alt="image" src="https://github.com/user-attachments/assets/0e8b63f7-efdc-4912-9bf0-c5e21b7dc04d">

The Python issue is probably separate.

**Comment by Cidree** [comment]
That's great to hear @jmcphers ! 
Shall we close this issue, or keep it with the Python `geemap` issue?

**Comment by juliasilge** [comment]
@Cidree can you create a Python mapping example that doesn't use Google Cloud Earth Engine? We aren't set up to use that proprietary service. Please do feel free to open a followup issue with that!

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2024.07.0-107
    OS Version          : OSX

### Test scenario(s) 

library(mapview)
mapview()

- then scrolling around/zooming in mapview

Link(s) to TestRail test cases run or created: 
N/A
